### PR TITLE
Map `f1_elc_op_mnt_expn` table

### PR DESCRIPTION
**Pull request by aesharpe**
This one was harder to map because the `row_literal` values differed more year to year than the other tables. But it wasn't too crazy.

Noteworthy things: 

- A blank `row_literal` for row 84 that I labeled as `row_type=header`
Screen Shot 2022-12-07 at 5 56 13 PM

- One `dbf_only=True` where `row_literal` was `(561) Load Dispatching` in 1994 and was broken into `(561.1) Load Dispatch-Reliability, (561.2)`,  `Load Dispatch-Monitor and Operate Transmission System`, and `(561.2) Load Dispatch-Monitor and Operate Transmission System` in later years of dbf and in xbrl. 

- A couple very close calls with naming. Ex: the transmission overhead line expense row literal maps to `overhead_line_expense` and the distribution overhead line expense row literal maps to `overhead_line_expenses`. Oye. 

- Some unmapped `xbrl_factoid` rows pertaining to energy storage. Seems like they are so new they aren't even in the 2022 FERC Form 1 PDF?: 
   - `maintenance_of_energy_storage_equipment`
   - `maintenance_of_energy_storage_equipment_other_power_generation`
   - `maintenance_of_energy_storage_equipment_transmission`
   - `operation_of_energy_storage_equipment`
   - `operation_of_energy_storage_equipment_distribution`
   - `operation_of_energy_storage_equipment_transmission_expense`
   - `power_purchased_for_storage_operations`

Here's a snapshot of the 2022 FERC Form 1 PDF with no mention of the `maintenance_of_energy_storage_equipment_transmission` field. In the xbrl taxonomy it is labeled: `570.1 Maintenance of Energy Storage Equipment`.

Screen Shot 2022-12-07 at 6 00 27 PM

**How should I deal with these unmapped xbrl factoids?**

### CODE REVIEW

**Comment by aesharpe** [comment]
> All of the `dbf_only = TRUE` rows need to have an `xbrl_factoid` value created for them -- a reasonable placeholder that can play the same role as the `xbrl_factoids` do in aligning the old DBF row numbers with each other when they have changed. Any row that does not have an `xbrl_factoid` value (e.g. the header rows) will be dropped from the DBF data, which isn't what we want to have happen, this applies to the old row 84 which was associated with FERC Account 561.

Thanks, I think I just forgot to do it here

File: src/pudl/package_data/ferc1/dbf_to_xbrl.csv [hunk 1]
```diff
@@ -1585,7 +1585,6 @@ f1_edcfu_epda,112,1994,Line 112,,,,
 f1_edcfu_epda,113,1994,Line 113,,,,
 f1_edcfu_epda,114,1994,Line 114,,,,
 f1_edcfu_epda,115,1994,Line 115,,,,
-f1_elc_op_mnt_expn,171,2006,TOTAL Customer Service and Information Expenses (Total 167 thru 170),calculated_value,customer_service_and_information_expenses,,
```
Comment (zaneselvans) [inline src/pudl/package_data/ferc1/dbf_to_xbrl.csv]: I moved this record to where it should be in the ordered table. Not sure why it showed up here.

File: src/pudl/package_data/ferc1/dbf_to_xbrl.csv [hunk 2]
```diff
@@ -1635,7 +1634,7 @@ f1_elc_op_mnt_expn,46,1994,(537) Hydraulic Expenses,reported_value,hydraulic_exp
 f1_elc_op_mnt_expn,47,1994,(538) Electric Expenses,reported_value,electric_expenses_hydraulic_power_generation,,
 f1_elc_op_mnt_expn,48,1994,(539) Miscellaneous Hydraulic Power Generation Expenses,reported_value,miscellaneous_hydraulic_power_generation_expenses,,
 f1_elc_op_mnt_expn,49,1994,(540) Rents,reported_value,rents_hydraulic_power_generation,,
-f1_elc_op_mnt_expn,50,1994,TOTAL Operation (Enter Total of Lines 44 thru 49),reported_value,hydraulic_power_generation_operations_expense,,
+f1_elc_op_mnt_expn,50,1994,TOTAL Operation (Enter Total of Lines 44 thru 49),calculated_value,hydraulic_power_generation_operations_expense,,
```
Comment (zaneselvans) [inline src/pudl/package_data/ferc1/dbf_to_xbrl.csv:1637]: Looked like this was miscategorized.

File: src/pudl/package_data/ferc1/dbf_to_xbrl.csv [hunk 3]
```diff
@@ -1669,7 +1668,7 @@ f1_elc_op_mnt_expn,80,1994,"TOTAL Power Production Expenses (Total of lines 21,
 f1_elc_op_mnt_expn,81,1994,2. TRANSMISSION EXPENSES,header,,,
 f1_elc_op_mnt_expn,82,1994,Operation,header,,,
 f1_elc_op_mnt_expn,83,1994,(560) Operation Supervision and Engineering,reported_value,operation_supervision_and_engineering_electric_transmission_expenses,,
-f1_elc_op_mnt_expn,84,1994,(561) Load Dispatching,reported_value,,TRUE,"A one to many row literal that is broken into (561.1) Load Dispatch-Reliability, (561.2) Load Dispatch-Monitor and Operate Transmission System, and (561.2) Load Dispatch-Monitor and Operate Transmission System in later years of dbf and in xbrl"
+f1_elc_op_mnt_expn,84,1994,(561) Load Dispatching,reported_value,load_dispatching_transmission_expense,TRUE,"A one to many row literal that is broken into (561.1) Load Dispatch-Reliability, (561.2) Load Dispatch-Monitor and Operate Transmission System, and (561.2) Load Dispatch-Monitor and Operate Transmission System in later years of dbf and in xbrl"
```
Comment (zaneselvans) [inline src/pudl/package_data/ferc1/dbf_to_xbrl.csv:1671]: Tried to come up with a reasonable category name here.

**Review by zaneselvans — APPROVED** [review]

**Pull request merged**